### PR TITLE
taking values from doc

### LINKED ISSUE #208: Market commentary block : take key value from doc

### PR DESCRIPTION
**Pull request by ravverma**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #208 

Test URLs:
- Before: https://main--icicidirect--aemsites.hlx.live/research/equity
- After: https://commentary-fix--icicidirect--aemsites.hlx.live/research/equity

image

### CODE REVIEW

File: blocks/market-commentary/market-commentary.js [hunk 1]
```diff
@@ -25,10 +25,10 @@ function createMarketCommentaryCard(cardData, placeholders) {
         div(
           { class: 'info' },
           p(
-            span(`${placeholders.powerby}`),
+            span(`${blockConfig.powerby}`),
```
Comment (anagarwa) [inline blocks/market-commentary/market-commentary.js:28]: @ravverma this is repeated in many blocks , IMO we should take it from placeholder

File: blocks/market-commentary/market-commentary.js [hunk 2]
```diff
@@ -25,10 +25,10 @@ function createMarketCommentaryCard(cardData, placeholders) {
...
           ),
           p(
-            span(`${placeholders.publishedon} ${NEWS_DATE}`),
+            span(`${blockConfig.publishedon} ${NEWS_DATE}`),
```
Comment (anagarwa) [inline blocks/market-commentary/market-commentary.js:31]: @ravverma this is repeated in many blocks , IMO we should take it from placeholder

File: blocks/market-commentary/market-commentary.js (hunk 1 above)
Comment (ravverma) [reply blocks/market-commentary/market-commentary.js:28]: I though it was for this only but I have checked it is using several places. reverting this change.

File: blocks/market-commentary/market-commentary.js (hunk 2 above)
Comment (ravverma) [reply blocks/market-commentary/market-commentary.js:31]: same as above

**Review by anagarwa — APPROVED** [review]

**Pull request merged**